DataMerge.ipynb

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
<p>Instead of <code>for n in graph.nodes: node = graph.nodes[n]</code>  you can do <code>for node in graph.nodes.values(): ...</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laitalaj/railanalysis/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>